eng/common/instructions/azsdk-tools/check-package-validation.instructions.md

```diff
@@ -0,0 +1,38 @@
+---
+description: 'Check if the package pr checks and/or validation has passed for the SDK package.'
+---
+## Goal
+Check the PR and Validation checks for the SDK package by collecting the required information from the user and executing the run_check command.
+
+## Instructions
+1. **Collect Required Information**:
+    - Prompt the user for the exact package name
```

Comment:
Do we want to prompt the user for the package path instead of the package name?

Reply:
I had it as package name because if we want multiple languages to be run the package path would not be the same

Reply:
But aren't the package names also a little different for each language too?